service/src/main/java/bio/terra/tanagra/service/accesscontrol/model/impl/AouWorkbenchAccessControl.java

```diff
@@ -133,10 +133,12 @@ public Set<Action> getStudyPermissions() {
         return null;
       }
     }
+
+    HttpStatus statusCode = (HttpStatus) responseEntity.getStatusCode();
     LOGGER.error(
         "AoU Workspace Access Level api error. Http status code: {}, Message: {}",
-        responseEntity.getStatusCodeValue(),
-        responseEntity.getStatusCode().getReasonPhrase());
+        statusCode,
```

Comment:
This looks like it's logging a different property -- is the change getStatusCodeValue > getStatusCode intentional?

Reply:
getStatusCodeValue() is deprecated in favor of getStatusCode(). Both return the same effectiv value
getStatusCodeValue --> int value of the enum, which is converted to string in Logger.error. 
getStatusCode -> httpStatus enum, which is converted to string in Logger.error. 

IMO enum to string is cleaner than int to string since the former leverages the native tostring implementation (if any) of the enum.